### PR TITLE
Update modals.md

### PR DESCRIPTION
**Pull request by code-a-man**
**Please describe the changes this PR makes and why it should be merged:**

add "Responding to modal submissions" as other interactions have already

### CODE REVIEW

File: guide/interactions/modals.md [hunk 1]
```diff
@@ -96,7 +96,25 @@ client.on('interactionCreate', interaction => {
 	console.log(interaction);
 });
 ```
+## Responding to modal submissions
 
+The `MessageComponentInteraction` class provides the same methods as the `CommandInteraction` class. These methods behave equally:
```
Comment (megatank58) [inline guide/interactions/modals.md:101]:
```suggestion
The `ModalSubmitInteraction` class provides the same methods as the `CommandInteraction` class. These methods behave equally:
```

File: guide/interactions/modals.md [hunk 2]
```diff
@@ -96,7 +96,25 @@ client.on('interactionCreate', interaction => {
...
+- `fetchReply()`
+- `deleteReply()`
+- `followUp()`
+
+```js{1,4-6}
+client.on('interactionCreate', interaction => {
+	if (!interaction.isModalSubmit()) return;
+
+	if (interaction.customId === 'myModal') {
+		await interaction.reply({ content: 'Your submit recieved!' });
```
Comment (megatank58) [inline guide/interactions/modals.md:114]:
```suggestion
		await interaction.reply({ content: 'Your submission was recieved successfully! });
```
The other sections do not check for customId so the surrounding if statement might not be consistent?

Comment (megatank58) [reply guide/interactions/modals.md:114]: I mean as in the other sections in the modal page

Comment (code-a-man) [reply guide/interactions/modals.md:114]: Hmm I don't know, maybe they need too

**Pull request merged**